crates/rolldown/tests/esbuild/default/export_wildcard_fs_node_common_js/artifacts.snap

```diff
@@ -8,7 +8,6 @@ snapshot_kind: text
 
 ```js
 "use strict";
-require("./external");
 
 //#region internal.js
 let foo = 123;
```

Comment:
<!--__GRAPHITE_HTML_TAG_START__--><i class='graphite__hidden'>[Re: line +25]<br/><br/></i>

<!--__GRAPHITE_HTML_TAG_END__-->Use this as an example, `./external` is already imported here, and the first line `require` is generated because we treat `export * from 'external'` as a `imports_from_external_module`<!--__GRAPHITE_HTML_TAG_START__--><span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/rolldown/rolldown/2946?utm_source=unchanged-line-comment">Graphite</a>.</span>

<!--__GRAPHITE_HTML_TAG_END__-->